### ISSUE TITLE
Run on Windows?

### ISSUE THREAD

**Issue by ngr-t** [jlegewie/sublime-paramiko#1]
Hello, thank you for maintaining this dependency.

I want to use paramiko from [my package](https://github.com/ngr-t/SublimeHermes) on Windows.
However, it is registered for [OSX only](https://github.com/wbond/package_control_channel/blob/341d3bf2419b3215d52ed43520e87b333b7ea83b/repository/dependencies.json#L519) so we can't use it on Windows.
I wonder why it is, because you have already omitted all non-Python dependencies from the original implementation, so it should work on all platform (sorry I have not confirmed yet, though).
Doesn't this run on Windows? Just not confirmed?

Thanks

**Comment by jlegewie** [comment]
Hi, I don't have a windows machine and never tested it. So it might work...

**Comment by ngr-t** [comment]
OK. I have Windows and Linux machines. Would you like me to test it? When it was confirmed to run on Windows and Linux, could you (or can I?) change the platform of this dependency as 'all'?
I think we can use the test suite of original paramiko implementation.

**Comment by jlegewie** [comment]
Yes, you can just sent a pull request but please confirm that it works first.

**Comment by null-directory** [comment]
I'm not sure which package is requiring paramiko on Windows (well, pyzmq but I don't know what asked for it), but it causes Package Control to throw an error popup every start. Since it's marked only for OSX, it never resolves.

May I test this somehow to verify it works in Windows? Is there a way I can force it to be loaded? (Having a long delayed modal popup interrupt fire every time Sublime opens is bizarrely frustrating!)

**Comment by Aethylred** [comment]
This is still outstanding,

```
Package Control: Installing 1 missing dependency
Package Control: The dependency "paramiko" is either not available on this platform or for this version of Sublime Text
Package Control: Installed missing dependency paramiko
Package Control: Skipping automatic upgrade, last run at 2019-04-08 10:52:38, next run at 2019-04-08 11:52:38 or after
error: Package Control

1 missing dependency was just installed. Sublime Text should be restarted, otherwise one or more of the installed packages may not function properly.
```

Which results in a dialog box opening _every_ time Sublime Text is started in Windows.

**Comment by sschuhmann** [comment]
I would be able to test this package on a windows machine. Therefore I would need the steps how I can install it. As ngr-t mentioned, we could use the build in test suite, I just have problems to install it properly
